site/blog/series-a-announcement.md

```diff
@@ -0,0 +1,109 @@
+
+Architectures like agents, RAGs, and most recently MCP (Model Context Protocol) have expanded what's possible with AI, but they've also dramatically increased the attack surface:
+
+- **Autonomous agents** execute actions with production system access
+- **RAG pipelines** can leak entire knowledge bases through retrieval manipulation
+- **MCP servers** enable tools and APIs that introduce new injection vectors
+- **Multimodal systems** accept text, voice, and images—each a potential attack vector
+
+As these architectures hit production, we're seeing real-world incidents of prompt injection, data leakage, insecure tool use, and more. Traditional security tools weren't built for this complexity, and manual testing can't keep pace.
+
+<!-- truncate -->
+
+## Who's using Promptfoo
+
+The adoption has been incredible:
+
+- **125,000+ developers** have downloaded our open source tools
+- **30+ Fortune 500 companies** run Promptfoo in production CI/CD pipelines
+- **Foundation model labs** test their models with Promptfoo before release
+
+## Welcome to Ganesh Bell
+
+We're excited to have Ganesh Bell, Managing Director at Insight Partners, join our board. He brings deep experience scaling developer platforms at GE Digital and Uptake.
+
+"Promptfoo has created a category-defining product," said Ganesh Bell, Managing Director at Insight Partners. "Every serious enterprise deploying AI will need comprehensive security testing, and Promptfoo's combination of technical depth, open source adoption, and enterprise traction positions them perfectly to capture this massive market opportunity. We're thrilled to partner with them as they build the full AI security stack."
+
+"When we first invested in Promptfoo, we recognized that AI security would become mission-critical for every enterprise," said Zane Lackey, General Partner at Andreessen Horowitz. "In just one year, they've validated that thesis and then some. This additional investment reflects our conviction that they'll define how companies approach AI security for years to come."
+
+## What's next: Building for the agentic future
+
+The internet is being rebuilt for agents, and the APIs they call are increasingly being exposed as Model Context Protocol (MCP) servers. We're seeing a shift from single-turn text to long-running, tool-using agents that can collaborate, hand off tasks, and critique each other. These agents may live in untrusted environments, execute code, and have access to sensitive data. Large enterprises are embedding vast amounts of proprietary data into RAGs and agents. These new architectures introduce unique risks.
+
+Our roadmap focuses on four key trends:
+
+**First**, we're adding deep support for agentic and multi-agent workflows, including chain-level tracing and safety checks that watch conversations unfold over many steps.
```

Comment:
Don't bold these - if you have to bold then highlight the important bit in the line